src/test/java/page/googlecloud/GoogleCloudPricingCalculatorPage.java

```diff
@@ -0,0 +1,97 @@
+package page.googlecloud;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class GoogleCloudPricingCalculatorPage {
+
+	@FindBy(css = "[ng-model$='quantity']")
+	public WebElement inputNumberOfInstances;
+
+	@FindBy(css = "iframe[src*=calculator]")
+	public WebElement calculatorIFrame;
+
+	@FindBy(id = "myFrame")
+	public WebElement myFrame;
+
+	@FindBy(css = "md-select[ng-change*=Series]")
+	public WebElement seriesSelectField;
+
+	@FindBy(css = "md-option[value=n1]")
+	public WebElement seriesSelectedOptions;
+
+	@FindBy(css = "md-select[ng-change*='Compute']")
+	public WebElement machineTypeSelectedField;
+
+	@FindBy(css = "md-option[value$=STANDARD-8]")
+	public WebElement machineTypeSelect;
+
+	@FindBy(css = "div[class=md-label]")
+	public WebElement addGPUCheckbox;
+
+	@FindBy(css = "md-select[ng-model$=gpuCount]")
+	public WebElement numberOfGPUDropdown;
+
+	@FindBy(css = "div.md-clickable md-select-menu md-content md-option[value=\"1\"]")
```

Comment:
It's not a good practice to keep hardcoded locators. What if I want to choose an option with value=2? I suggest not to keep it as a separate element, but have a string with a generic locator

Reply:
Dima, it was a locator for element which one we were finding togather :) only this one is working there